src/python/pants/backend/python/subsystems/ipython.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List
+
+from pants.backend.python.subsystems.python_tool_base import PythonToolBase
+
+
+class IPython(PythonToolBase):
+  options_scope = 'ipython'
+  default_version = 'ipython==5.8.0'
```

Comment:
This is the last version to support Python 2. Technically, we shouldn't care about this anymore now that Pants defaults to only working with Python 3 (e.g. defaults to Pytest 5.0). But, that is a breaking change that we should go through a deprecation cycle for, so I punted on the issue.

Either way, this is a big upgrade from us previously defaulting to 1.0.0.